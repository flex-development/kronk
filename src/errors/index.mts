/**
 * @file Entry Point - Errors
 * @module kronk/errors
 */

export { default as CommandError } from '#errors/command.error'
export { default as KronkError } from '#errors/kronk.error'
