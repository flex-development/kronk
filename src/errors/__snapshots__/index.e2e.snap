// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`e2e:errors > should expose public api 1`] = `
[
  "CommandError",
  "KronkError",
]
`;
